BuildResidentialHPXML/measure.rb

```diff
@@ -1970,6 +1970,12 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setDescription("If the distribution system is #{HPXML::DHWDistTypeRecirc}, the recirculation pump power. If not provided, the OS-HPXML default is used.")
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeIntegerArgument('hot_water_distribution_recirc_num_units_served', false)
```

Comment:
Maybe this should now be `xxx_num_bedrooms_served` so we continue to follow analogous water heater arguments...